bokeh/util/compiling.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import absolute_import
```

Comment:
`compile` conflicts with Python's `compile()`, so perhaps `compiler`. `jsbuild` is wrong as well, because you can do more than compiling JavaScript in this module.